lua/rest-nvim/request/init.lua

```diff
@@ -66,8 +66,8 @@ local function get_body(bufnr, start_line, stop_line, has_json)
     if line:find("{%%") then
       break
     end
-    -- Ignore commented lines with and without indent
-    if not utils.contains_comments(line) then
+    -- Ignore commented lines with and without indent and Ignore forms
+    if not utils.contains_comments(line) and not line:find("^-F .+") then
```

Comment:
we may not want to add those to body but we surely want to replace variables, dont we ?